client/src/components/graph/graph.js

```diff
@@ -188,16 +143,30 @@ class Graph extends React.Component {
       responsive,
       selectionTool,
       currentSelection,
-      layoutChoice
+      layoutChoice,
+      graphInteractionMode
     } = this.props;
-    const { reglRender, mode, regl, svg } = this.state;
+    const { reglRender, regl, svg } = this.state;
     let stateChanges = {};
 
-    if (reglRender && this.reglRenderState === "rendering" && mode !== "zoom") {
+    if (
```

Comment:
these two if statements (this one and the next one) are almost identical, except for the test of `graphInteractionMode`.   They are quite hard to parse as written.   Suggest you restructure as:
* outside if that tests for initialization state (if reglRender && this.reglRenderState === "rendering")
* inside if that branches on `graphInterationMode`
That makes the actual tests much clearer.

Reply:
No, that's not quite correct. The inner loop has two conditionals: `=== zoom` & `!== zoom`. I can break out reglRender, however!

Reply:
"If it's rendering and it's not zoom, stopped rendering. If it isn't rendering and it is zoom, start rendering."